Answers.md

```diff
@@ -0,0 +1,10 @@
+1. Semantic HTML is a way to make the anchors and tags of your website more "descriptive" so it's meta-data can be more easily found and comprehended. Instead of just <div> everywhere, semantic tags like <header> <section> <article> <footer> <figure> more accurately describe the "meaning" of your content, not just what goes where. 
+
+2. The main difference is, an element with {display: block} takes up the entire width of the browser - or as much space as possible; and {display: inline} only takes up the space of the content (i.e., text or image) contained in that element. 
+
+3. The four areas of the box model include: Content, Padding, Border, and Margin. 
+
+4. The property {align-items: center} is using the 'row' or horizontal axis {flex-direction: row}. 
+
```

Comment:
Cross-axis is the correct phrase here. The cross axis can change based on with flex-direction you are doing, so I see what you were getting at. Think about it like if your do `flex-direction: row` then your cross-axis is vertical, if you do `flex-direction: column` your cross axis is horizontal